### PR TITLE
compose - add container click and set focus on chatbox

### PR DESCRIPTION
**Pull request by susanm74**
fixes https://github.com/nyaruka/temba-components/issues/274 and also fixes https://github.com/nyaruka/temba-components/issues/292

### CODE REVIEW

**Comment by susanm74** [comment]
@ericnewcomer i'm not sure about the single random unrelated field manager (will investigate as best i can), but besides that, this is RFR

**Comment by susanm74** [comment]
@ericnewcomer this is RFR, but can you sanity check my fix for that broken temba-field-manager component filter test?

File: src/fields/FieldManager.ts [hunk 1]
```diff
@@ -304,6 +305,7 @@ export class FieldManager extends StoreElement {
         placeholder="Search"
         @change=${this.handleSearch}
         clearable
+        value=${this.query}
```
Comment (susanm74) [inline src/fields/FieldManager.ts:308]: @ericnewcomer i noticed that the field-manager wasn't updating the inner text-input component, so i added the value attribute, and that seemed to help fix things

File: test/temba-field-manager.test.ts [hunk 2]
```diff
@@ -29,8 +30,11 @@ describe('temba-field-manager', () => {
   it('filters', async () => {
     await loadStore();
     const fm: FieldManager = await getEle();
-    (fm.shadowRoot.querySelector('#search') as HTMLDivElement).click();
-    await type('at');
+    (fm.shadowRoot.querySelector('#search') as TextInput).focus();
```
Comment (susanm74) [inline test/temba-field-manager.test.ts:33]: @ericnewcomer i noticed that the search text-input wasn't being styled correctly so i switched from an HTMLDivElemet to a TextInput and switched from a click() to a focus()

File: test/temba-field-manager.test.ts [hunk 3]
```diff
@@ -29,8 +30,11 @@ describe('temba-field-manager', () => {
...
+    fm.query = 'at';
+    await fm.updateComplete;
```
Comment (susanm74) [inline test/temba-field-manager.test.ts:35]: @ericnewcomer i noticed that the search `temba-textinput` wasn't being updated with the query... so i switched from an `await type('at');` to `fm.query = 'at';` and added an `fm.updateComplete`

File: screenshots/truth/list/fields-filtered.png
Comment (ericnewcomer) [inline screenshots/truth/list/fields-filtered.png:1]: Why did this screenshot grow?

**Review by ericnewcomer — APPROVED** [review]

File: screenshots/truth/list/fields-filtered.png
Comment (susanm74) [reply screenshots/truth/list/fields-filtered.png:1]: @ericnewcomer i'm still not sure why this sometimes happens 😬

**Pull request merged**